dxr/static/js/dxr.js

```diff
@@ -426,23 +391,30 @@ $(function() {
                     return;
                 }
                 data.query = query;
-                // New results, overwrite
+                // New results, display them.
                 if (myRequestNumber > displayedRequestNumber) {
                     displayedRequestNumber = myRequestNumber;
-                    populateResults(data, false);
-                    var pushHistory = function() {
-                        history.pushState({}, '', queryString);
+                    populateResults(data, appendResults);
+                    var pushHistory = function () {
```

Comment:
I didn't mean you had to change it as part of this PR; I just wondered if you saw anything I didn't. It's better not to change it (or at least to keep it as a separate commit).